diff -urN aaa/devtodo-0.1.14/src/todoterm.cc devtodo-0.1.14/src/todoterm.cc
--- aaa/devtodo-0.1.14/src/todoterm.cc	Thu Nov  8 11:51:20 2001
+++ devtodo-0.1.14/src/todoterm.cc	Wed Jul 24 23:42:58 2002
@@ -4,8 +4,8 @@
 #ifdef USETERMCAP
 #include <string>
 #include <stdexcept>
-#include <curses.h>
-#include <term.h>
+#include <ncurses/curses.h>
+#include <ncurses/term.h>
 
 static char info[2048];
 
